go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -25,7 +23,14 @@ const (
 	defaultOpenFlowCookie = "0xdeff105"
 )
 
-func addService(service *kapi.Service, inport, outport, gwBridge string, nodeIP *net.IPNet) {
+type nodePortWatcher struct {
+	inport   string
```

Comment:
inport and outport here are kind of ambiguous from the previous addService function call. Maybe we should name them something more obvious like ofPhysPort and ofPatchPort

Reply:
Renamed to `ofPhysPort` and `ofPatchPort`